bookings/components/booking-filters.yaml

```diff
@@ -0,0 +1,464 @@
+actual_guests:
+  name: actual_guests
+  in: query
+  required: false
+  schema:
+    type: string
+  description: >-
+    Filter by the number of `actual_guests`.
+
+    You can filter by a range of actual guests, eg `1-10` for any bookings with an `actual_guest`
+    between 1 and 10. You can also pass through more than one ranges, for example, `1-4,10-15` will 
+    bring back bookings between 1 to 4 *and* between 10 to 15.
+  example: 1-10
+assigned_area:
+  name: assigned_area
+  in: query
+  required: false
+  explode: false
+  schema:
+    type: array
+    items:
+      type: string
+  description: >-
+    Filters on bookings that are assigned to the given area ID. If a comma-separated list is passed, 
+    bookings assigned to any of the given area IDs will be returned. Alternatively, to filter on 
+    whether bookings have an assigned area or not, use `true` or `false`. 
+  example: 5cab10d8f44d3861c8c3819d
+assigned_to:
+  name: assigned_to
+  in: query
+  required: false
+  explode: false
+  schema:
+    type: array
+    items:
+      type: string
+  description: >-
+    Filters on bookings that are assigned to the given user ID. If a comma-separated list is passed, 
+    bookings assigned to any of the given user IDs will be returned. To filter on bookings that are 
+    not assigned to a user, use `none`.
+  example: 5cab111386510d011b597b3f
+reference:
+  name: reference
+  in: query
+  required: false
+  schema:
+    type: integer
+  example: 12345
+  description: >-
+    Filters on bookings that have the given booking reference ID.
+created_date:
+  name: created_date
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-01-01'
+  description: >-
+    Filters on bookings that were created on the given date.
```

Comment:
Do we also need to link to a section on standard date filters (eg `2019-01-01 TO *`) or would you prefer to get people using `_from` and `_to`?

Reply:
I don't think we need to mention the fact that we support to ` TO ` syntax since the `_from` and `_to` is more explicit and performs the same function, and nothing is gained by documenting the alternative syntax